test/shopify-cli/theme/dev_server/app_extension_proxy_test.rb

```diff
@@ -207,46 +252,8 @@ def test_hop_to_hop_headers_are_removed_from_proxied_response
           end
         end
 
-        def test_pass_pending_templates_to_storefront
-          skip # TODO: remove skip once replace_templates param is formed properly
-          ShopifyCLI::DB
-            .stubs(:get)
-            .with(:shop)
-            .returns("dev-theme-server-store.myshopify.com")
-
-          ShopifyCLI::DB
-            .stubs(:get)
-            .with(:storefront_renderer_production_exchange_token)
-            .returns("TOKEN")
-
-          stub_request(:post, "https://dev-theme-server-store.myshopify.com/?_fd=0&pb=0")
-            .with(
-              body: {
-                "_method" => "GET",
-                "replace_templates" => {
-                  "layout/theme.liquid" => @theme["layout/theme.liquid"].read,
-                },
-              },
-              headers: {
-                "Accept-Encoding" => "none",
-                "Authorization" => "Bearer TOKEN",
-                "Content-Type" => "application/x-www-form-urlencoded",
-                "Cookie" => "_secure_session_id=#{SECURE_SESSION_ID}",
-                "Host" => "dev-theme-server-store.myshopify.com",
-                "X-Forwarded-For" => "",
-                "User-Agent" => "Shopify CLI",
-              }
-            )
-            .to_return(status: 200, body: "PROXY RESPONSE")
-
-          stub_session_id_request
-          response = request.get("/")
-
-          assert_equal("PROXY RESPONSE", response.body)
-        end
-
         def test_patching_store_urls
-          skip # TODO: remove skip once replace_templates param is formed properly
+          skip # not sure how this test should work
```

Comment:
same here -- need some advice on how this test should work

Reply:
I believe we can just remove the `test_patching_store_urls` from this class, for now.

A similar test is passing in the `theme_proxy_test.rb` class, but I believe it's actually a good idea to create a `proxy_test.rb` for this (I will take care of creating a task to follow up this) ☺️